### ISSUE TITLE
Pyside2 and exec()

### ISSUE THREAD

**Issue by PeterJust** [pyzo/pyzo#775]
Dear Developers,

after the recent Arch package update to 4.11.4 the interpreter crashes on startup giving the following traceback:
```
Python 3.9.9 (main, Nov 20 2021, 21:30:06) on linux (64 bits).
Traceback (most recent call last):
This is the Pyzo interpreter with integrated event loop for PYSIDE2.
Type 'help' for help, type '?' for a list of *magic* commands.
  File "/usr/lib/python3.9/site-packages/pyzo/pyzokernel/start.py", line 159, in <module>
    __pyzo__.run()
  File "/usr/lib/python3.9/site-packages/pyzo/pyzokernel/interpreter.py", line 224, in run
    self.guiApp.run(self.process_commands, self.sleeptime)
  File "/usr/lib/python3.9/site-packages/pyzo/pyzokernel/guiintegration.py", line 578, in run
    exec_ = getattr(self._QtGui.real_QApplication, "exec")
AttributeError: type object 'PySide2.QtWidgets.QApplication' has no attribute 'exec'

The kernel process exited. (1)
```
Apparently this is caused by pyside2 and can be prevented by replacing "exec()" by "exec_()". https://stackoverflow.com/questions/62916408/pyside2-qtwidgets-qapplication-object-has-no-attribute-exec

Is this fixed already in 4.11.7 and I just have to bump my package maintainer to update the package, or is this news to you?

Best,
Peter

**Comment by almarklein** [comment]
Yes, this should be fixed in 4.11.7. It's actually the only change compared to 4.11.6.

**Comment by PeterJust** [comment]
Ok, thanks!
